bitrise.yml

```diff
@@ -1542,8 +1542,9 @@ workflows:
             $HOME/google-cloud-sdk/bin/gcloud version
             $HOME/google-cloud-sdk/bin/gcloud auth activate-service-account -q --key-file /tmp/key-file.json
             $HOME/google-cloud-sdk/bin/gcloud config set project "$GOOGLE_CLOUD_PROJECT"
-            $HOME/google-cloud-sdk/bin/gcloud firebase test ios run --test /Users/vagrant/git/Fennec_Enterprise.zip \
-                --model=iphone14pro,version=16.4 \
+            $HOME/google-cloud-sdk/bin/gcloud firebase test ios run \
+                --test /Users/vagrant/git/Fennec_Enterprise.zip \
+                --device model=iphone14pro,version=16.4 \
```

Comment:
I don't think this version is available on Firebase...
![Screenshot 2023-09-19 at 18 32 33](https://github.com/mozilla-mobile/firefox-ios/assets/1897507/e8768a94-920b-4fb8-be92-db4d8e3f8c64)

Reply:
oh god, you're right. just updated it. good catch!